addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/tests.html

```diff
@@ -76,6 +76,31 @@ <H2>YAML</H2>
   
 </pre>
 
+<H2>Url Presence</H2>
+
+Presence tests are supported by the activeScan and passiveScan-wait jobs. These tests can be used to validate the
```

Comment:
I dont think this is true is it? They should be supported by at least all of the explore jobs, and I'd be fine for them to supported by all jobs.